Vercel and Prisma updates (#1473)
neondatabase/website: content/docs/guides/prisma.md

## Use connection pooling with Prisma

> Serverless functions can require a large number of database connections as demand increases. If you use serverless functions in your application, it is recommended that you use a pooled Neon connection string, as shown:

Reviewer: ```suggestion
Serverless functions can require a large number of database connections as demand increases. If you use serverless functions in your application, we recommend that you use a pooled Neon connection string, as shown:
```

```ini shouldWrap
# Pooled Neon connection string
DATABASE_URL="postgres://alex:AbC123dEf@ep-cool-darkness-123456-pooler.us-east-2.aws.neon.tech/dbname?sslmode=require"
```

A pooled Neon connection string adds `-pooler` to the endpoint ID, which tells Neon to use a pooled connection. You can add `-pooler` to your connection string manually or copy a pooled connection string from the **Connection Details** widget on the Neon **Dashboard**. Use the **Pooled connection** checkbox to add the `-pooler` suffix.

### Connection pooling with Prisma Migrate

> Prior to Prisma ORM 5.10, attempting to run Prisma Migrate commands, such as `prisma migrate dev`, with a pooled connection causes the following error:

Reviewer: To make it grammatical. But maybe it would be better to phrase it as:
"If you are using Prisma ORM 5.10 or earlier, attempting to run Prisma Migrate commands, such as `prisma migrate dev`, with a pooled connection causes the following error:"

```ini shouldWrap
# Pooled Neon connection string
DATABASE_URL="postgres://alex:AbC123dEf@ep-cool-darkness-123456-pooler.us-east-2.aws.neon.tech/dbname?sslmode=require"

# Unpooled Neon connection string
DIRECT_URL="postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname?sslmode=require"
```

#### Using a pooled connection with Prisma Migrate

> With Prisma ORM 5.10 or higher, you can use a pooled Neon connection string with Prisma Migrate. In this case, you only need to define the pooled connection string in your `schema.prisma` file. Adding a `directUrl` property to the datasource block in your `schema.prisma` file and defining a `DIRECT_URL` setting in your environment file are not required. Your complete configuration will look like this:

Reviewer: Does the directURL part need to be mentioned at all? If it was a prior workaround that no longer applies, it's probably not needed now. But if it's a common configuration for other integrations though, it's good.

Author: yeah, it would be common for any using Prisma Migrate with Neon. I think we can leave it.
